### DISCUSSION TITLE
Air quality

### DISCUSSION THREAD

**Discussion by larryb31** [jaydeethree/Home-Assistant-weatherdotcom#10] [Ideas]
Hi, could we possibly get their air quality sensors, as they appear in their apps/web page?

**Comment by jaydeethree** [comment]
Unfortunately air quality information is not available in the APIs that are used by this integration. Here are the two APIs that are used and the data available in each:

[API for current weather conditions](https://docs.google.com/document/d/1UOjfN1j54Ocmwj-zYCcHG_RM1tlNceW_qlzz0wVILXA/edit)

[API for weather forecasts](https://docs.google.com/document/d/1RY44O8ujbIA_tjlC4vYKHKzwSwEmNxuGw5sEJ9dYjG4/edit)

The best options that I know about for air quality info are either the [AirNow](https://www.home-assistant.io/integrations/airnow/) or [IQVIA](https://www.home-assistant.io/integrations/iqvia/) integrations in Home Assistant.

**Comment by larryb31** [comment]
Ok, thanks for the info.
